hedera-mirror-rest/tokens.js

```diff
@@ -344,23 +347,122 @@ const getAndValidateTokenIdRequestPathParam = (req) => {
   return EntityId.fromString(tokenIdString, constants.filterKeys.TOKENID).getEncodedId();
 };
 
+/**
+ * Validates the token info timestamp filter operator.
+ *
+ * @param {string} op
+ * @return {boolean}
+ */
+const isValidTokenInfoTimestampFilterOp = (op) => {
+  return op === 'eq' || op === 'lt' || op === 'lte';
+};
+
+/**
+ * Validates token info request filters. Only timestamp filter is supported.
+ *
+ * @param param
+ * @param op
+ * @param val
+ * @return {boolean}
+ */
+const validateTokenInfoFilter = (param, op, val) => {
+  let ret = false;
+
+  if (op === undefined || val === undefined) {
+    return ret;
+  }
+
+  if (param === constants.filterKeys.TIMESTAMP) {
+    ret = isValidTokenInfoTimestampFilterOp(op) && utils.isValidTimestampParam(val);
+  }
+
+  return ret;
+};
+
+/**
+ * Transforms the timestamp filter operator: '=' maps to '<=', others stay the same
+ *
+ * @param {string} op
+ */
+const transformTimestampFilterOp = (op) => {
+  const {opsMap} = utils;
+  return op === opsMap.eq ? opsMap.lte : op;
+};
+
+/**
+ * Gets the token info query
+ *
+ * @param {string} tokenId
+ * @param {[]} filters
+ * @return {{query: string, params: []}} the query string and params
+ */
+const extractSqlFromTokenInfoRequest = (tokenId, filters) => {
+  const conditions = ['token_id = $1'];
```

Comment:
Use Model so we can avoid rewriting column names be hand. 
```suggestion
  const conditions = [`${Token.TOKEN_ID = $1`];
```